database/src/main/kotlin/no/nav/su/se/bakover/database/behandling/BehandlingRepoInternal.kt

```diff
@@ -21,12 +21,12 @@ import java.util.UUID
 
 internal object BehandlingRepoInternal {
     fun hentBehandling(behandlingId: UUID, session: Session): Behandling? =
-        "select * from behandling where id=:id"
+        "select * from behandling where id=:id and avsluttetBegrunnelse is null"
```

Comment:
Er det aldri slik at vi skal ha muligheten til å hente ut disse? Burde det ikke synes at man har "avsluttet/henlagte/lukkede" behandlinger for saksbehandler? Evt filtrerer dette i frontend? Evt legg noe "foran" repoet som filtrerer ut dette? Det at "avsluttetBegrunnese is null" må legges til i 10 forskjellige spørringer lukter vondt for meg. Hvis vi aldri har noe behov for å se disse igjen kan vi vel like gjerne bare slette dem?

Reply:
Det blir vel åpen for diskusjon egentlig. jeg endte opp med å lande på at det skal bli mulig en eller annen dag å hente dem ut på en måte, og enn så lenge, bare filtere dem ut. Kan flytte dette til front-end.